custom_components/omnik_inverter/manifest.json

```diff
@@ -9,6 +9,6 @@
     "@robbinjanssen",
     "@klaasnicolaas"
   ],
-  "requirements": ["omnikinverter==0.7.0"],
+  "requirements": ["git+https://github.com/MarijnS95/python-omnikinverter@tcp#omnikinverter==0.8.0"],
```

Comment:
Please only bump the version

Reply:
I'm sure I added a comment here; this will be bumped as soon as the TCP backend is merged and released, but for now it allows others to easily test this PR if they wish.